backend/tests/integration-async/migrator/test_migrator_tasks.py

```diff
@@ -662,3 +661,219 @@ def test_update_relations_with_modified_experiments(
     ).one()
 
     assert experiment_literature_relation.literature_uuid == record.id
+
```

Comment:
you could add app_config fixture and add `config["FEATURE_FLAG_ENABLE_AUTHOR_DISAMBIGUATION"] = True`